### PR TITLE
fix Ui changes as per figma design

### PR DESCRIPTION
**Pull request by abbasali1994**
Ticket(s):
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/issues/1925
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/issues/1926
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
   h4,
   h5,
   h6 {
-    @apply font-heading;
+    @apply font-bold;
```
Comment (ap-justin) [inline src/index.css:13]:
```suggestion
    @apply font-heading font-bold;
```
should not remove `font-heading`, many `<h>` titles are in `montserrat`

also if we are going to abstract `font-bold` here, be sure to remove `font-bold` in all `<h>` that have this className

**Review by ap-justin — DISMISSED** [review]

File: src/index.css (hunk 1 above)
Comment (abbasali1994) [reply src/index.css]: Done with the requested change @ap-justin , apologies , i wasn't aware clicking the commit suggestion auto-resolved the comment

Comment (misicnenad) [reply src/index.css]: The changes were already committed at the time of this comment being resolved, so all was according to protocol.
@ap-justin try reloading the PR files page next time, it happens that GH doesn't show the most recent changes.

File: src/components/donation/Steps/Donater/Form/index.tsx [hunk 2]
```diff
@@ -50,20 +50,20 @@ export default function Form(props: {
       )}
 
       <div
-        className={`flex ${
+        className={`flex gap-6 ${
```
Comment (misicnenad) [inline src/components/donation/Steps/Donater/Form/index.tsx:53]: **nit:**
```suggestion
        className={`flex gap-3 md:gap-5 ${
```
As expected per Figma:
- [desktop link](https://www.figma.com/file/sDjoeJ2IFPd9EJ1LI2WMvF/Product-Design?node-id=1024-46929&t=n6YhtRrqWsgjocnp-0)
image
- [mobile link](https://www.figma.com/file/sDjoeJ2IFPd9EJ1LI2WMvF/Product-Design?node-id=1024-48277&t=n6YhtRrqWsgjocnp-0)
image

**Review by misicnenad — CHANGES_REQUESTED** [review]

**Comment by misicnenad** [comment]
@abbasali1994 once conflicts are resolved and [change request ](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1953#discussion_r1144561715)addressed, we can merge.

File: src/index.css (hunk 1 above)
Comment (ap-justin) [reply src/index.css]: > @ap-justin try reloading the PR files page next time, it happens that GH doesn't show the most recent changes.

@misicnenad, what do you mean? I've re-requested changes 20 hours ago, 
image

while the changes regarding this were just made 6 hours ago
image

Comment (abbasali1994) [reply src/index.css]: @ap-justin you are correct in pointing that out, i was working on these changes as well but i made the mistake of accepting "commit suggestion" which prematurely resolved the ticket, no harm done , i hv done the changes now

File: src/components/donation/Steps/Donater/Form/index.tsx (hunk 2 above)
Comment (abbasali1994) [reply src/components/donation/Steps/Donater/Form/index.tsx]: Done

**Comment by abbasali1994** [comment]
> e

Rebased and added changes

File: src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx [hunk 3]
```diff
@@ -29,7 +29,7 @@ export default function AdminLinks(props: WalletState) {
 
   return (
     <div className="grid p-4 gap-3 border-b border-prim">
-      <h3 className="font-heading font-bold text-sm text-gray-d1 dark:text-gray">
```
Comment (misicnenad) [inline src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx:32]: Need to remove `font-heading` and `font-bold` from these components as well:
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/ConnectedWallet/Details/MobileTitle.tsx#L8
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/KadoModal.tsx#L38
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/ConnectedWallet/Details/Favourites/index.tsx#L16
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/WalletSelectorOpener/WalletModal.tsx#L15
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/contexts/ErrorContext/InstallWalletContent.tsx#L25
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/pages/Admin/Charity/WidgetConfigurer/WidgetConfigurer.tsx#L7

**Review by misicnenad — CHANGES_REQUESTED** [review]

File: src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx (hunk 3 above)
Comment (abbasali1994) [reply src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx:32]: i will do it

Comment (abbasali1994) [reply src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx:32]: @misicnenad Done, thanks for finding these file , i mainly used regular expressions to find <h> tags with these classnames

**Review by SovereignAndrey — APPROVED** [review]

**Review by misicnenad — APPROVED** [review]

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**